tasks/lib/compress.js

```diff
@@ -54,6 +55,12 @@ module.exports = function(grunt) {
       + grunt.util.pluralize(files.length, 'file/files.'));
   };
 
+  exports.brotli = function(files, done) {
+      exports.singleFile(files, iltorb.compressStream, 'br', done);
+      grunt.log.ok('Compressed ' + chalk.cyan(files.length) + ' '
+          + grunt.util.pluralize(files.length, 'file/files.'));
```

Comment:
I just realized that travis allows to configure C++ compiler support in .travis.yml. Will add the later today too.